docs/devreference/devref.rst

```diff
@@ -0,0 +1,76 @@
+.. _developerreference:
+
+Developer Reference
```

Comment:
I think most of this is now duplicated from the getting started guide... Maybe just make it link?